app/src/main/res/layout/activity_main.xml

```diff
@@ -1,6 +1,22 @@
 <?xml version="1.0" encoding="utf-8"?>
 <androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    android:layout_width="match_parent"
-    android:layout_height="match_parent">
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_height="match_parent"
+    android:layout_width="match_parent">
+
+    <androidx.fragment.app.FragmentContainerView
+        android:id="@+id/fragment_container"
+        android:name="com.example.plantsapp.TasksFragment"
+        android:layout_width="match_parent"
```

Comment:
Inside of constraint layout we don't use match_parent - instead of this we use 0dp and set constraints correctly 

So remember this and fix in all  layouts